Solutions/PaloAlto-PAN-OS/Playbooks/PlaybooksOnPrem/PaloAlto-PAN-OS-BlockURL/azuredeploy.json

```diff
@@ -0,0 +1,1229 @@
+                        "Configured_address_group": {
+                           "description": "compose predefined address group",
+                           "inputs": "@body('List_address_groups')?['result']?['entry']?[0]?['@name']",
+                           "runAfter": {
+                              "Set_variable_address_group_members": [
+                                 "Succeeded"
+                              ]
+                           },
+                           "type": "Compose"
+                        },
+                        "Filter_array_URL_from_list_of_security_rules": {
+                           "description": "This filters all the security rules in which this URL is a member",
+                           "inputs": {
+                              "from": "@body('List_security_rules')?['result']?['entry']",
+                              "where": "@contains(item()?['destination']?['member'], items('For_each_malicious_URL')?['Url'])"
+                           },
+                           "runAfter": {
+                              "Configured_address_group": [
+                                 "Succeeded"
+                              ]
+                           },
+                           "type": "Query"
+                        },
+                        "Filter_array_of_URL_address_from_list_of_address_objects": {
+                           "description": "This filters the list of address objects in which this URL is a member ",
+                           "inputs": {
+                              "from": "@body('List_address_objects')?['result']?['entry']",
+                              "where": "@equals(item()?['fqdn'], items('For_each_malicious_URL')?['Url'])"
+                           },
+                           "runAfter": {
+                              "Set_variable_adaptive_card_body": [
+                                 "Succeeded"
+                              ]
+                           },
+                           "type": "Query"
+                        },
+                        "List_address_groups": {
+                           "description": "This gets complete list of address object groups present in the PAN-OS",
+                           "inputs": {
+                              "host": {
+                                 "connection": {
+                                    "name": "@parameters('$connections')['PaloAltoConnector']['connectionId']"
+                                 }
+                              },
+                              "method": "get",
+                              "path": "/restapi/v10.0/Objects/AddressGroups",
+                              "headers": {
+                                 "X-PAN-KEY": "@body('Get_secret')?['value']"
+                              },
+                              "queries": {
+                                 "location": "vsys",
+                                 "name": "AddressGroup",
```

Comment:
Hard coded value used , which is impacting the logic app run .

Value expected here is actually address group